src/style.css

```
@@ -0,0 +1,121 @@
+@import url(https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap);
+
+* {
+  box-sizing: border-box;
+}
+
+.card-body {
```

```suggestion
.body {
```

the class for `body` has no relation to the card 😉

---

okey :)